homeassistant/components/lutron_caseta/device_trigger.py

```diff
@@ -429,9 +456,9 @@ async def async_get_triggers(
 
 def _device_model_to_type(device_registry_model: str) -> str:
     """Convert a lutron_caseta device registry entry model to type."""
-    model, p_device_type = device_registry_model.split(" ")
+    *model_list, p_device_type = device_registry_model.split(" ")
     device_type = p_device_type.replace("(", "").replace(")", "")
-    return _lutron_model_to_device_type(model, device_type)
+    return _lutron_model_to_device_type(" ".join(model_list), device_type)
```

Comment:
In the device registry, the model is a combination of {model} ({device_type})
PJ2-3BRL-XXX-L01 (Pico3ButtonRaiseLower)

The model and device_type are extracted from the brackets and the spacing.
However, homeowner keypads return "Homeowner Keypad" as the model, adding an extra space.
Homeowner Keypad (HomeownerKeypad)

This extracts the device_type for the last word in the sentence only, and re-combines all the previous words to re-form the model.